test/sets.jl

```diff
@@ -216,6 +216,13 @@ u = unique([1,1,2])
 @test @inferred(unique(x for x in 1:1)) == [1]
 @test unique(x for x in Any[1,1.0])::Vector{Real} == [1]
 
+# unique!
+@testset "unique!" begin 
```

Comment:
There's a trailing space after `begin`, which is causing `make check-whitespace` to fail.